pkg/controller/installation/products/codeready/reconciler.go

```diff
@@ -24,9 +25,9 @@ import (
 
 const (
 	defaultInstallationNamespace = "codeready-workspaces"
-	defaultClientName            = "che-client"
```

Comment:
Just to confirm, this should have always been called `codeready-client`?

Reply:
Good question. I can change it to codeready for consistency.